client/apollo/react/src/Form/TextArea/TextAreaCommon.tsx

```diff
@@ -74,23 +70,24 @@ const TextArea = forwardRef<HTMLTextAreaElement, Props>(
         />
         <textarea
           id={inputId}
-          className={componentClassName}
+          className="af-form__input-textarea"
           ref={inputRef}
-          aria-errormessage={error ? idError : undefined}
-          aria-describedby={helper ? idHelp : undefined}
-          aria-invalid={Boolean(error)}
-          {...otherProps}
+          aria-errormessage={error ? errorId : undefined}
+          aria-describedby={helper ? helperId : undefined}
+          required={required}
+          aria-invalid={Boolean(error) || classModifier.includes("error")}
```

Comment:
```suggestion
          aria-invalid={Boolean(error) || classModifier.includes("error") ? true :  undefined}
```

aria less is best ...

Reply:
Fixed in https://github.com/AxaFrance/design-system/pull/1385/commits/b436fe23c6c1f7ec5c778187fa753fab805979ee